src/utils/prepareResults.ts

```diff
@@ -0,0 +1,48 @@
+const md5 = require('blueimp-md5');
+
+import { uniqBy } from 'lodash';
+
+import { compareResults } from '../formatters/utils/sortResults';
+import { IRuleResult } from '../types';
+
+export type ComputeFingerprintFunc = (rule: IRuleResult, hash: (val: string) => string) => string;
+
+export const defaultComputeResultFingerprint: ComputeFingerprintFunc = (rule, hash) => {
+  let id = String(rule.code);
+
+  if (rule.path && rule.path.length) {
```

Comment:
```suggestion
  if (rule.path.length) {
```
`IRuleResult` enforces `path` to be defined.